lib/Word_Lib/Word_Lemmas_Internal.thy

```diff
@@ -838,4 +838,19 @@ lemma ucast_ucast_mask_id:
    UCAST('b \<rightarrow> 'a) (UCAST('a \<rightarrow> 'b) (x && mask n)) = x && mask n" for x :: "'a::len word"
   by (simp add: ucast_ucast_len[OF eq_mask_less])
 
+lemma uint_less_1_eq:
```

Comment:
the name sounds a bit strange, like uint of something less than 1, or the other meaning where "less 1 " means subtracting 1 is even more confusing... it should be in the family of `uint_less_le_eq`, so `uint_less_1_le_eq` maybe? `uint_minus_1_less_le_eq` is a bit too much.